services/brig/docs/swagger-internal-endpoints.md

```diff
@@ -5,3 +5,8 @@ between services and site operators for forensics.
 
 Request execution does not work as Swagger expects a single target host whereas
 these endpoints are served by multiple hosts.
+
+Displayed paths are prefixed by the service name (e.g. `/<brig>`). This is
+necessary because Swagger does not expect more than one endpoint for a given
+path and method. Though, some paths (e.g. `/i/status`) exist on more than one
+service. To use the a path, remove the prefix.
```

Comment:
Alternatively, we could expose the internal swagger UI for each service on their own page?

Is it possible to use the swagger UI to call those endpoints, the way it is implemented now?

Reply:
Thanks for the inspiration! I was more focused on the overview over all endpoints. However, being able to use the endpoints is pretty appealing, too.

As this should be possible with a `kubectl port-forward`, I'll give it a shot.

Reply:
maybe different PR, though?

Reply:
> maybe different PR, though?

@fisx , Sounds good :+1: